doc/modules/clustering.rst

```diff
@@ -1801,10 +1801,10 @@ pairwise precision and recall:
 Where ``TP`` is the number of **True Positive** (i.e. the number of pair
 of points that belong to the same clusters in both the true labels and the
 predicted labels), ``FP`` is the number of **False Positive** (i.e. the number
-of pair of points that belong to the same clusters in the true labels and not
-in the predicted labels) and ``FN`` is the number of **False Negative** (i.e. the
-number of pair of points that belongs in the same clusters in the predicted
-labels and not in the true labels).
+of pair of points that belong to the same clusters in the predicted labels and not
+in the true labels) and ``FN`` is the number of **False Negative** (i.e. the
+number of pair of points that belong to the same clusters in the true
+labels and not in the predicted labels).
```

Comment:
Maybe we can use bulletpoints and a bit of rephrasing for clarity here. Something similar to:
> - True Positive (TP): Pairs of points that are clustered together both in the ground truth and in the predicted clustering.
> - False Positive (FP): Pairs of points that are clustered together in the predicted clustering but not in the ground truth.
> - False Negative (FN): Pairs of points that are clustered together in the ground truth but not in the predicted clustering.